app/services/foods_api/api_calls.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'endpoints'
+
+module ApiCalls
+  extend Endpoints
+
+  def search_by_food(food)
+    response = get find_foods_endpoint, params: {
+      query: food
+    }
+    response[:results]
```

Comment:
In this particular api, the food data that you're looking for is nested under the key of `:foods`.

Reply:
Also, since you have symbolized the json data at this point, you won't be able to call `response[:foods]`, you'll have to call `response['foods']`